src/utils/graphql/queries/getAllResponses.query.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getAllResponses($serial: String!) {
+    applicationBySerial(serial: $serial) {
+      applicationResponses {
```

Comment:
fingers crossed, last thing.

Was testing with a couple of buttons that mutate response (that changes other response to visible/invisible). Was failing to reload + the network thing was failing as per:

> can't currently test that the hook re-evalutes correctly when responses change, so please check that the logic is correct. The useEffect hook is currently only watching "data" -- is this correct?

After adding id and serial (for application) here, the network-only policy worked and query hook was re-running and visibility validation was work : )

Reply:
Awesome, thanks 343ac7e